launcher/modplatform/flame/FlameCheckUpdate.cpp

```diff
@@ -140,15 +140,14 @@ void FlameCheckUpdate::executeTask()
         setStatus(tr("Parsing the API response from CurseForge for '%1'...").arg(mod->name()));
 
         if (!latest_ver.has_value() || !latest_ver->addonId.isValid()) {
-            emit checkFailed(mod, tr("No valid version found for this mod. It's probably unavailable for the current game "
-                                     "version / mod loader."));
+            emit checkFailed(mod, tr("No valid version found for this mod."));
             continue;
         }
 
         if (latest_ver->downloadUrl.isEmpty() && latest_ver->fileId != mod->metadata()->file_id) {
             auto pack = getProjectInfo(latest_ver.value());
             auto recover_url = QString("%1/download/%2").arg(pack.websiteUrl, latest_ver->fileId.toString());
-            emit checkFailed(mod, tr("Mod has a new update available, but is not downloadable using CurseForge."), recover_url);
+            emit checkFailed(mod, tr("Cannot download update due to author restrictions."), recover_url);
```

Comment:
```suggestion
            emit checkFailed(mod, tr("Unable to update mod due to author restrictions."), recover_url);
```

Keeping context in this error message doesn't make much sense after removing the above -- though I'm definitley glad it's still here 😄 

I do wonder though: are "author restrictions" really the only reason standard API downloads from Curseforge would be blocked? In the case they aren't, I would rather keep the original message here as this could be misleading

Reply:
The are only three cases where a download link for a mod update would be missing that I can think of

1) author has blocked 3rd party clients for the mod that it was previously enabled for
2) author or curseforge has pulled the the upload for this version
3) the mod has updated but the new version doesn't support this game version (pretty sure we filter this out before )

Reply:
I actually like the suggestion. It is verbally shorter, and sounds a bit less in-your-face.

And my concern with the original "Curseforge restrictions" is what I had explained previously; it incorrectly implies that it is Curseforge itself that is preventing any mod from being updated externally. The pointed phrasing there can induce panic and confusion, especially with how inconsistent it would sound. While there was a bit of a scare in that direction, after concessions it is a setting used by users, either intentionally, or an infrequent mod author who was unaware of automatic "defaults" that were there at one point of time.